firestore/mock_test.go

```diff
@@ -187,6 +187,27 @@ func (s *mockServer) RunQuery(req *pb.RunQueryRequest, qs pb.Firestore_RunQueryS
 	return nil
 }
 
+func (s *mockServer) RunAggregationQuery(req *pb.RunAggregationQueryRequest, qs pb.Firestore_RunAggregationQueryServer) error {
+	res, err := s.popRPC(req)
+	if err != nil {
+		return err
+	}
+	responses := res.([]interface{})
+	for _, res := range responses {
+		switch res := res.(type) {
+		case *pb.RunAggregationQueryResponse:
+			if err := qs.Send(res); err != nil {
```

Comment:
`return qs.Send(res)`

With a return in each branch, you can also remove the `return nil` outside of the statement.

Reply:
Compiler doesn't like it when I remove the `return` statement, even when I add a `return nil` after the `if` in the first branch.